<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hakai Kelp-O-Matic 9000</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_image_files">
      <property name="text">
       <string>Image Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_image_files"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_clear_images">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_open_images">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_output_directory">
      <property name="text">
       <string>Output directory</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_output_directory</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_output_directory" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_output_directory"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_output_directory">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_output_filename_pattern">
      <property name="text">
       <string>Output filename pattern</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_output_filename_pattern</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_output_filename_pattern">
      <property name="text">
       <string>{basename}_kelp.tif</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_classification_model">
      <property name="text">
       <string>Classification model</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_classification_model</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_classification_model"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_advanced_options">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Crop dimension</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_batch_size">
         <property name="text">
          <string>Batch size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_batch_size">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string>2^</string>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_crop_padding">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Crop padding</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="spinBox_crop_dimension">
         <property name="minimum">
          <number>16</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_cuda">
         <property name="text">
          <string>Use CUDA</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Accelerator</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_show_advanced_options">
         <property name="text">
          <string>Show advanced options</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_run">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_progress" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Files processed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_overall">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_current_file">
         <property name="text">
          <string>Current file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_img">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_output_directory</tabstop>
  <tabstop>toolButton_output_directory</tabstop>
  <tabstop>lineEdit_output_filename_pattern</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_show_advanced_options</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_advanced_options</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
